app/scripts/modules/cloudfoundry/src/pipeline/stages/createServiceKey/cloudfoundryCreateServiceKeyStage.module.ts

```diff
@@ -1,6 +1,6 @@
 import { CloudfoundryCreateServiceKeyStageConfig } from './CloudfoundryCreateServiceKeyStageConfig';
 import { ExecutionDetailsTasks, IStage, Registry } from '@spinnaker/core';
-import { CloudfoundryCreateServiceKeyExecutionDetails } from './CloudfoundryCreateServiceKeyExecutionDetails';
+import { CloudfoundryServiceKeyExecutionDetails } from 'cloudfoundry/presentation';
```

Comment:
We can talk about this tomorrow. Is this kind of self-referential import OK in every circumstance? I really don't know.

Reply:
It appears that Amazon's code has had references like this for a couple of years:  https://github.com/spinnaker/deck/blob/0a5fd58ed76c894f9a8b0ab7027bd07e0dfc43ce/app/scripts/modules/amazon/src/loadBalancer/TargetGroup.tsx#L7
and we have had them for at least a couple of months elsewhere: for example, https://github.com/spinnaker/deck/blob/0a5fd58ed76c894f9a8b0ab7027bd07e0dfc43ce/app/scripts/modules/cloudfoundry/src/pipeline/stages/destroyService/cloudfoundryDestroyServiceStage.module.ts#L3